[INFO] Using pretrained UNet from segmentation_models_pytorch.
Epoch 1/20 - Loss: 0.7951
Validation - Dice: 0.7813, IoU: 0.6519
Epoch 2/20 - Loss: 0.4470
Validation - Dice: 0.8239, IoU: 0.7143
Traceback (most recent call last):
  File "/home/neo/amFOSS/AI_Projects/Polyp_Segmentation/sweep.py", line 2, in <module>
    from train import train
  File "/home/neo/amFOSS/AI_Projects/Polyp_Segmentation/train.py", line 96, in <module>
    outputs = model(images)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/segmentation_models_pytorch/base/model.py", line 66, in forward
    features = self.encoder(x)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/segmentation_models_pytorch/encoders/efficientnet.py", line 77, in forward
    x = block(x, drop_connect_prob)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/segmentation_models_pytorch/encoders/_efficientnet.py", line 176, in forward
    x = self._bn1(x)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/modules/batchnorm.py", line 193, in forward
    return F.batch_norm(
           ~~~~~~~~~~~~^
        input,
        ^^^^^^
    ...<9 lines>...
        self.eps,
        ^^^^^^^^^
    )
    ^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/nn/functional.py", line 2831, in batch_norm
    torch.backends.cudnn.enabled,
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/torch/backends/__init__.py", line 39, in __get__
    return self.getter()
           ~~~~~~~~~~~^^
KeyboardInterrupt
Exception ignored in atexit callback <function _start_and_connect_service.<locals>.teardown_atexit at 0x771c088d84a0>:
Traceback (most recent call last):
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/wandb/sdk/lib/service_connection.py", line 90, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/wandb/sdk/lib/service_connection.py", line 218, in teardown
    self._router.join()
  File "/home/neo/amFOSS/venv/lib/python3.13/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/usr/lib/python3.13/threading.py", line 1092, in join
    self._handle.join(timeout)
KeyboardInterrupt:
